README.md

```diff
@@ -353,6 +354,12 @@ possible security implications.
 | :----------: | ------------------------------------------------------------- |
 | turbo        | Turbo frequencies are enabled in Intel pstate driver
 
+### System Features
+
+| Feature     | Attribute        | Description                                 |
+| ----------- | ---------------- | --------------------------------------------|
+| cpu_power   | sst_bf.enabled   | Speed Select Technology - Base frequency is enabled in cpu           |
```

Comment:
Move under `cpu` source. I think feature could be `power` and attribute as is `sst_bf.enabled`.

Reply:
+1, will change it.